default.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    ":combinePatchMinorReleases",
+    ":dependencyDashboard",
+    ":pinAllExceptPeerDependencies",
+    ":separateMajorReleases",
+    "config:base",
+    "group:monorepos",
+    "schedule:weekends",
+    "github>bitwarden/renovate-config:pin-actions"
+  ],
```

Comment:
❓ Some missing ones here that were in the template:

- `pinAllExceptPeerDependencies`: Because it's NPM-specific?
- `rebaseStalePrs`: Why not?

Reply:
We have `pinAllExceptPeerDependencies`.

Reply:
Weird, not sure how I missed that.